client/landing/jetpack-cloud/components/date-picker/style.scss

```diff
@@ -1,32 +1,58 @@
 .date-picker {
 	display: flex;
 	align-items: center;
-	max-width: 30rem;
 	background: #fff;
 	padding-top: 0.5rem;
 	font-size: 16px;
+
+	@include breakpoint( '>660px' ) {
```

Comment:
I think it would be good to use the Gutenberg mixin here. (@import '~@wordpress/base-styles/breakpoints';)

Reply:
I'm going to propose it as a task to be applied to the JP Cloud project. Thanks